### DISCUSSION TITLE
RSS FEED as a link

### DISCUSSION THREAD

**Discussion by RogueGhost93** [FreshRSS/FreshRSS#5923] [Q&A]
Is it possible to have RSS Feed as a link instead of dowloaded file of all of my news in freshrss? To integrate it with self hosted apps like Homarr.
 Thanks

**Comment by Alkarex** [comment]
I have not understood the question.
Can you try to reformulate? Maybe giving examples

**Comment by math-GH** [comment]
Try right click on the RSS icon and chose "copy link"

**Comment by RogueGhost93** [comment]
For example, lets say you have RSS Feed App that can import RSS only as links, how would i go about adding all of my rss feeds from freshrss to this app as a single link, not as a separate link for each news source but as 1 link for all of my news sources from freshrss combined? I hope this this makes it little clearer.  Something like https://freshrss.mydomain.com/feed

**Reply by Frenzie** [reply]
I believe that's what @math-GH was referring to:
![image](https://github.com/FreshRSS/FreshRSS/assets/202757/45d2c5be-9993-41b7-87af-e85ade009710)
